data_models/asana_data_model.py

```diff
@@ -0,0 +1,16 @@
+import panther_event_type_helpers as event_type
+
+
+def get_event_type(event):
+    audit_log_type_map = {
+        "user_login_succeeded": event_type.SUCCESSFUL_LOGIN,
+        "user_login_failed": event_type.FAILED_LOGIN,
+        "user_invited": event_type.USER_ACCOUNT_CREATED,
+        "user_reprovisioned": event_type.USER_ACCOUNT_CREATED,
+        "user_deprovisioned": event_type.USER_ACCOUNT_DELETED,
+        "user_workspace_admin_role_changed": event_type.ADMIN_ROLE_ASSIGNED,
+    }
```

Comment:
In the atlassian data model you initialized this at the global level (just above this method), any reason for the difference?

Reply:
Ah no I moved it trying to debug the thing we were talking about. I'll move it back